lib/new_relic/agent/instrumentation/sinatra.rb

```diff
@@ -32,18 +32,29 @@
   end
 
   executes do
-    if Sinatra::Base.respond_to?(:build)
```

Comment:
Would you please let us know about why we don't have to worry about this check anymore?

Reply:
The `:build` method was added in Sinatra 1.2.1. The agent stopped supporting versions below 1.4.x in version 6.15.0. This condition will never be false for the versions we support.